main.tf

```diff
@@ -100,6 +100,15 @@ resource "aws_security_group" "ecs" {
 resource "aws_ecs_cluster" "default" {
   name = var.name
   tags = var.tags
+
+  capacity_providers = aws_ecs_capacity_provider.default[*].name
```

Comment:
I believe this will break if `capacity_provider_asg_arn` is not set...

Reply:
Nope, it works without it =)

<img width="1092" alt="afbeelding" src="https://user-images.githubusercontent.com/7990862/127455019-ad28f1c8-5ee6-4543-b407-078a8a9b6933.png">

Reply:
Cool 👍🏻  So only comment left about this, is that should be:

```
  name               = var.name
  capacity_providers = aws_ecs_capacity_provider.default[*].name
  tags               = var.tags
```

Reply:
Done